### ISSUE TITLE
dashboard link tile overhaul

### ISSUE THREAD

**Issue by ezeek** [ezeek/smartly#62]
northan- having the ability to make dashboard link tiles look more tile-like, maybe using a large (taking up most of the tile) customizable icon and customizable title in smaller text above or below it.

**Comment by ezeek** [comment]
```
.tile.dashboard {
    height: 100%;
}

#tile-2 .tile-primary {
    visibility: hidden;
}

#tile-2 .tile-primary:before {
    display: block;
    line-height: 1.2em;
    font-size: 40px;
    visibility: visible;
}

#tile-2 .tile-primary:after {
    font-size: 18px;
    margin-left: 0px;
    line-height: 1.2em;
    /* display: none; */
}

**Comment by ezeek** [comment]
released with v1.08
